src/main/java/org/traccar/notificators/NotificatorTelegram.java

```diff
@@ -93,12 +95,17 @@ private LocationMessage createLocationMessage(Position position) {
 
     @Override
     public void sendSync(long userId, Event event, Position position) {
+        User user = Context.getPermissionsManager().getUser(userId);
         TextMessage message = new TextMessage();
-        message.chatId = chatId;
+        if (user.getAttributes().containsKey("notificationTelegramChatId")) {
```

Comment:
The key seems to be too long. We can probably cut it down to just `telegramChatId`.

Reply:
Would that be the only comment?

Reply:
I think so. Otherwise looks fine to me. One other minor thing is that I would probably just call
```
user.getString("notificationTelegramChatId")
```
instead of checking `containsKey` first.

Reply:
Done.